lab/graph_builder.py

```diff
@@ -36,5 +37,5 @@ def add_bar(self, label, val):
         self.bars.append((label, val))
```

Comment:
You're right, since we already have an _xlabels_ variable lets just go ahead and append a value to keep track of. Let's also change the name of `self.bars` to `self.bar_vals` to be more accurate.